src/main/java/com/dreammaster/gthandler/nameRemover/NameRemover.java

```diff
@@ -81,20 +81,35 @@ public int checkRecipe() {
                 case 2:
                     removeDisassembly = true;
                     break;
+                case 3:
+                    removeColor = true;
+                    break;
+                case 4:
+                    removeRepair = true;
+                    break;
                 default:
                     removeName = true;
                     removeDisassembly = true;
+                    removeColor = true;
+                    removeRepair = true;
             }
 
             if (removeName && nbt.hasKey("display")) {
                 nbt.getCompoundTag("display").removeTag("Name");
+                nbt.getCompoundTag("display").removeTag("color");
```

Comment:
hmm, this won't properly remove display.color if you pass in `removeColor`.  Should change the top level check to `hasKey(display)`, and then each individual check for removeName, and removeColor